puppet/zulip/files/cron.d/downgrade-small-realms-behind-on-payments

```diff
@@ -0,0 +1,5 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+0 23 * * * zulip /home/zulip/deployments/current/manage.py downgrade_small_realms_behind_on_payments
```

Comment:
How did you pick this time?

Reply:
Choosing 23:00 was kind of arbitary. Invoice plans was was done on 22:00 for reference. But actually thinking about it a bit more I have updated the time to 17 instead. Stripe normally tries to attempt charge on the invoice instantly after it's finalized but I guess it doesn't hurt to keep a bit more buffer time in case the process gets delayed by a couple of hours at Stripe's end. Keeping it at 17 make sure we don't  downgrade realms with 2 open invoices but with second invoice yet to to be charged by Stripe.